backend/log.go

```diff
@@ -12,3 +26,22 @@ var Logger = log.DefaultLogger
 var NewLoggerWith = func(args ...interface{}) log.Logger {
 	return log.New().With(args...)
 }
+
+func withContextualLogAttributes(ctx context.Context, pCtx PluginContext, endpoint string) context.Context {
```

Comment:
FYI for core plugins using the sdk logger we need to provide these contextual attributes as well together with existing https://github.com/grafana/grafana/blob/50504ba00803a291ea24e8a4a48b14494b97c744/pkg/services/pluginsintegration/clientmiddleware/contextual_logger_middleware.go#L59C1-L69 which currently is onyl for the grafana logger :) For now I guess we can duplicate this code to populate attributes for both core grafana logger and sdk logger in core grafana?

Eventually we probably want to include tenant id as well, but we can open a separate issue for that.